src/lwip/freertos/sys_arch.c

```diff
@@ -88,7 +88,7 @@ err_t sys_sem_new(sys_sem_t * sem, u8_t count)
 void sys_sem_free(sys_sem_t * sem)
 {
     vSemaphoreDelete(*sem);
-    SYS_STATS_DEC(sem);
+    SYS_STATS_DEC(sem.used);
```

Comment:
where does this change come from? Why was it not needed for other FreeRTOS platforms?